main.cpp

```diff
@@ -84,42 +102,90 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
+void playGuitarNote(int string = 5, int fret = 4, float holdLength = .12f)
+{
+    ignoreUnused(string, fret, holdLength);
+}
 
 /*
  2)
  */
+int trumpetNote(bool valve1 = true, bool valve2 = false, bool valve3 = true, int blowStyle = 0)
+{
+    ignoreUnused(valve1, valve2, valve3, blowStyle);
+    return {};
+}
 
 /*
  3)
- */
-
-/*
- 4)
- */
+ */
+int getTempo(float note1StartPosition = 54.01f, float note2StartPosition = 54.4f, int intendedSubdivision = 16)
+{
+    ignoreUnused(note1StartPosition, note2StartPosition, intendedSubdivision);
+    return{};
+}
+
+/*
+4)
+*/
+int selectMenuItem(int daysSinceLastPizza = 6, int daysSinceLastSub = 3, bool addSalad = true)
+{
+    ignoreUnused(daysSinceLastPizza, daysSinceLastSub, addSalad);
+    return{};
+}
 
 /*
  5)
  */
+float overdriveSetting(float distortionAmount = 3.2f, float tone = 7.66f, float gain = 11.f)
+{
+    ignoreUnused(distortionAmount, tone, gain);
+    return{};
+}
 
 /*
  6)
  */
+bool didTheCatKnockOverTheTrash(bool trashEverywhere = true, bool dogFilthy = false, bool catFilthy = true)
+{
+    ignoreUnused(trashEverywhere, dogFilthy, catFilthy);
+    return {};
+}
 
 /*
  7)
  */
+void clickMouse(bool rightHanded = true, int mouseButton = 0, int xValue = 459, int yValue = 744)
+{
+    ignoreUnused(rightHanded, mouseButton, xValue, yValue);
+}
 
 /*
  8)
  */
+double waterRemaining(double waterTotal = 1.0, double waterRemoved = .355577)
+{
+    ignoreUnused(waterTotal, waterRemoved);
+    return{};
+}
 
 /*
  9)
  */
+bool changeBattery(int daysSinceLastChange = 200, float averageHoursUsePerDay = 4.2f)
+{
+    ignoreUnused(daysSinceLastChange, averageHoursUsePerDay);
+    return{};
+}
 
 /*
  10)
  */
+char chordRoot(char note1 = 'C', char note2 = 'A', char note3 = 'E')
```

Comment:
This function name does not conform to the programming guideline:
```
Name Things what they are.
Name Actions based on what they do
```

The name you chose reflects a property (noun) and does not contain a verb, which would reflect an action.
Functions perform work, so their names must contain a verb which describes the work being performed.
Examples of function names that contain verbs:
- `computeAValue()`
- `estimateDistanceTraveled()`
- `performDiagnostics()`

- [ ] revise